gammapy/catalog/gammacat.py

```diff
@@ -170,6 +391,19 @@ def _add_source_meta(self, table):
         m['reference_id'] = d['reference_id']
 
     @property
+    def _flux_points_table_formatted(self):
+        """Returns formatted version of self.flux_points.table"""
+        table = self.flux_points.table.copy()
+
```

Comment:
Again, please shorten this a bit (remove empty lines, use list comprehension for setting formats).